src/test/resources/pages/adminEmailCompose.html

```diff
@@ -347,7 +344,7 @@
                                 </span>
                             </form>
     
-                            <div id="documentBaseUrl">${app.url}</div>
+                            {*}
```

Comment:
Need to find out for sure if it can indeed be empty. Can't simply treat the symptoms, need to find out the cause :-)